### PR TITLE
Fixng a couple links

### PR DESCRIPTION
**Pull request by slorello89**

### CODE REVIEW

File: docs/about/license.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ aliases:
 Redis is **open source software** released under the terms of the **three clause BSD license**. Most of the Redis source code was written and is copyrighted by Salvatore Sanfilippo and Pieter Noordhuis. A list of other contributors can be found in the git history.
 
 The Redis trademark and logo are owned by Redis Ltd. and can be
-used in accordance with the [Redis Trademark Guidelines](/docs/project/trademark).
+used in accordance with the [Redis Trademark Guidelines](/docs/about/trademark/).
```
Comment (enjoy-binbin) [inline docs/about/license.md:14]: /docs/about/trademark/
i am worry about the last `/`, it seems to be no problem, do not know which is better

**Review by enjoy-binbin — APPROVED** [review]
for the records:
https://redis.io/docs/about/license/
https://redis.io/docs/manual/data-types/#geospatial-indexes

File: docs/about/license.md (hunk 1 above)
Comment (zuiderkwast) [reply docs/about/license.md:14]: @enjoy-binbin If you try https://redis.io/docs/about/trademark you can see that it redirects to https://redis.io/docs/about/trademark/ so I think it's better to use the latter.

(It's unfortunate that all the URLs have changed though. It'd have been better to keep the old URLs so we wouldn't need to update all the links...)

**Comment by itamarhaber** [comment]
@slorello89 care to rebase against master?

**Comment by nermiller** [comment]
Closing as these links have been fixed since.

**Pull request closed without merging**